src/core/common/message.cpp

```diff
@@ -98,7 +98,7 @@ ThreadError Message::Init(void)
     return kThreadError_None;
 }
 
-Message *Message::New(uint8_t aType, uint16_t aReserved)
+Message *Message::New(uint8_t aType, uint16_t aReserved, bool aLinkSecurityEnabled)
```

Comment:
Why add this here? Setting the link security with `SetLinkSecurity()` after it's been allocated doesn't seem unreasonable.